some wording changes and capitalization fixes

jenworthington opened the pull request.

jenworthington committed:
some wording changes and capitalization fixes

dtregonning commented on README.md:
```diff
@@ -2,7 +2,7 @@
 Splunk Connect for Docker is a plug-in that extends and expands Docker's logging capabilities so that customers can push their Docker and container logs to their Splunk on-premise or cloud deployment.
 
-Splunk Connect for Docker is a supported open source product. Customers with an active Splunk support contract receive Splunk Extension support under the Splunk Support Policy. Splunk Connect for Docker is not supported on windows. Please refer to Docker Engine managed plugin system documentation on support for Microsoft Windows and other platforms as well as Prerequisites in this document. 
+Splunk Connect for Docker is a supported open source product. Customers with an active Splunk support contract receive Splunk Extension support under the Splunk support policy. Splunk Connect for Docker is not supported on Windows. Refer to Docker Engine managed plugin system documentation on support for Microsoft Windows and other platforms and to Prerequisites in this document for more informat ...
```
> "Splunk Support Policy"  - i think this is a use of a capitalization because a document is a proper noun. 

> and to Prerequisites

 - doesn't flow

jenworthington committed:
Update README.md

Added a link for Splunk Support Policy document. Took another stab at supported systems references.

sharonx commented on README.md:
```diff
@@ -27,7 +27,7 @@ http://docs.splunk.com/Documentation/Splunk/7.0.3/Data/UseHECusingconffiles
 * Make sure that indexer acknowledgement is disabled for your token. 
 * Optionally, enable the indexer acknowledgement functionality by clicking the Enable indexer management checkbox. 
-* Do not generate your token using the default TLS cert provided by Splunk. For information about configuring Splunk to use self-signed or third-party certs, see http://docs.splunk.com/Documentation/Splunk/7.0.3/Security/AboutsecuringyourSplunkconfigurationwithSSL.
+* Do not generate your token using the default TLS cert provided by Splunk. The default certificates are not secure. For information about configuring Splunk to use self-signed or third-party certs, see http://docs.splunk.com/Documentation/Splunk/7.0.3/Security/AboutsecuringyourSplunkconfigurationwithSSL.
```
can we have the latest release link instead of 7.0.3?

dbaldwin-splunk replied:
Not all the docs are updated with 7.1 at release.  7.0.3 may be current for some time after Splunk Enterprise GA.

jenworthington committed:
Update README.md

dbaldwin-splunk reviewed (approved):
Valid addition as comments are in alignment with ProdSec.
